docs/features/if_statements.md

```diff
@@ -0,0 +1,32 @@
+# If Statements
+
+```js
+// Simple if statement that will always be false
+
+if (1 /= 1) { // usually if (1 != 1) in other langugaes
+	print("This will never run")
+} else if (false) {
+	print("This will never run")
+} else {
+	print("This will always run")
+}
+
+// The condutional let will only run the block of code if the value can be assigned to the variable
```

Comment:
Perhaps a better example could be used here that could feasibly sometimes but not always succeed/fail? (ie an enum or list)

Reply:
ah, just use the example below